template/fragments.go

```diff
@@ -49,17 +50,20 @@ var elmTemplates = langTemplates{
 	arrayShortOpen:  ` List`,
 	arrayShortClose: ``,
 	basic:           ` {{updateElmType .Type}}`,
-	comment:         ` {{elmComment .Comment}}`,
+	fieldDocComment: `{{elmMultilineComment .DocComment 1}}`,
```

Comment:
Should this have a space at the start like the previous `comment` field had to keep it consistent?

Reply:
First, this is a multiline comment that precedes the field, so it does not need a space.

Second, for the single-line comment (handled in the `fieldClose` fragment), the whitespace is now prepended by the `func lineComment` that implements it.